### PR TITLE
refine cloudshell user experience

### PR DESCRIPTION
**Pull request by yungez**
refine cloudshell user experience. now, the flow is keep same, but output to user will be: 
1. upload playbook  -- user will not see this output, cause this is verbose and hacky. this should be replaced by official cloudshell sdk after it released.
2. run ansible playbook  -- user will see this output in cloudshell terminal, since this info is what user would like to see.

### CODE REVIEW

File: src/cloudConsole.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { window, commands, MessageItem, OutputChannel } from 'vscode';
+import { window, commands, MessageItem, OutputChannel, Terminal, env } from 'vscode';
 import { AzureAccount, AzureSession } from './azure-account.api';
```
Comment (zikalino) [inline src/cloudConsole.ts]: shouldn't have just * from vscode?

File: src/cloudConsoleLauncher.ts [hunk 2]
```diff
@@ -251,16 +265,19 @@ export async function delay(ms: number) {
 	return new Promise<void>(resolve => setTimeout(resolve, ms));
 }
 
-export async function runInTerminal(accessToken: string, consoleUri: string) {
-	process.stdin.setRawMode!(true);
-	process.stdin.resume();
+export async function runInTerminal(accessToken: string, consoleUri: string, tempFile: string) {
+	if (tempFile) {
```
Comment (zikalino) [inline src/cloudConsoleLauncher.ts]: hmm... i wouldn't mind more comments here. do we really have to check if tempFile is not null here?
why should anyone call this function with tempFile == null

Comment (yungez) [reply src/cloudConsoleLauncher.ts]: tempfile= null is scenario where caller doesn't care of terminal status, they just trigger connect to terminal, then return

Comment (zikalino) [reply src/cloudConsoleLauncher.ts]: ah, ok, i understand now

**Pull request merged**